spec/lib/simple_metric/version_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe SimpleMetric do
```

Comment:
This dummy spec is going to be broken each time we have a new release. Let's remove it.